extensions/exposition/source/Gateway.ts

```diff
@@ -83,9 +83,10 @@ export class Gateway extends Connector {
       throw new http.MethodNotAllowed()
 
     const body = await request.parse()
+    const query = Object.fromEntries(request.locator.searchParams)
```

Comment:
It's not always a good idea to do 1:1 mapping between searchParams and an object as search params can have repeated queries (for example, to represent an array of value in a standard way).
Are we sure we never will want repeated parameters in the searchParams? I would rather keep `query` as standard `URLSearchParams` as it seems to be more flexible.

Reply:
Any operation in Toa accepts `Request` [type](https://github.com/toa-io/toa/blob/05b32ddf5add27690dc544c5acaea64b949a4cab/types/source/call.d.ts#L1). Exposition definitely transforms HTTP Request into operation Request (`body` goes to `input`, and query string parsed to `Query`).

There is no such thing as an arbitrary query string in the Exposition. Anything not matching `Query` type will not pass operation call's validation and result in `Bad Request`.